frontend/src/component/feature/Dependencies/AddDependencyDialogue.tsx

```diff
@@ -189,6 +190,8 @@ export const AddDependencyDialogue = ({
             secondaryButtonText='Cancel'
         >
             <Box>
+                <DependenciesUpgradeAlert/>
+                <br />
```

Comment:
Why do we use the `br` tag? Shouldn't spacing between elements be handled by styling?